packages/components/src/scroll-lock/stories/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { withState } from '@wordpress/compose';
```

Comment:
we can just use `useState` now  right? This was written before  React hooks :)

Reply:
Oh, sure thing! I wasn't sure if we could safely use hooks (based on the readme example)

<3